subsys/bluetooth/audio/bap_scan_delegator.c

```diff
@@ -1035,12 +1131,15 @@ static int bt_bap_scan_delegator_init(const struct device *unused)
 	/* Store the pointer to the first characteristic in each receive state */
 	scan_delegator.recv_states[0].attr = &bass_svc.attrs[3];
 	scan_delegator.recv_states[0].index = 0;
+	scan_delegator.recv_states[0].active = false;
 #if CONFIG_BT_BAP_SCAN_DELEGATOR_RECV_STATE_COUNT > 1
 	scan_delegator.recv_states[1].attr = &bass_svc.attrs[6];
 	scan_delegator.recv_states[1].index = 1;
+	scan_delegator.recv_states[1].active = false;
 #if CONFIG_BT_BAP_SCAN_DELEGATOR_RECV_STATE_COUNT > 2
 	scan_delegator.recv_states[2].attr = &bass_svc.attrs[9];
 	scan_delegator.recv_states[2].index = 2;
+	scan_delegator.recv_states[2].active = false;
```

Comment:
The receive states at `static`ally allocated, so the `.active` field is initialized to `0`/`false` already